src/main/kotlin/de/danielmaile/mpp/item/ListenerConverter.kt

```diff
@@ -46,12 +45,12 @@ class ListenerConverter : Listener {
         val content = inventory.contents
 
         for (i in content.indices) {
-            val itemStack = content[i] ?: continue
-            val type = fromTag(itemStack) ?: continue
-            val typeStack = type.getItemStack(itemStack.amount)
+            val inventoryItemStack = content[i] ?: continue
+            val item = ItemRegistry.getItemFromItemstack(inventoryItemStack) ?: continue
+            val itemStack = item.itemStack(inventoryItemStack.amount)
 
-            if (itemStack == typeStack) continue
-            inventory.setItem(i, typeStack)
+            if (inventoryItemStack == itemStack) continue // Probably not happening
```

Comment:
Why is this not happening? inventoryItemStack equals itemStack if they are similar. This is the case 99% of the time.